sdk/ai/azure-ai-projects/README.md

```diff
@@ -663,6 +663,126 @@ In additional, you might find helpful to see the tracing logs in console.   You
+
+```python
+# Creating project client
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+
+# Upload data for evaluation and get dataset id
+data_id, _ = project_client.upload_file("<data_file.jsonl>")
+```
+
+##### [Optional] Azure OpenAI Model
+
+Azure AI project comes with a default Azure Open AI endpoint which can be easily accessed using following code. This gives you the endpoint details for you Azure OpenAI endpoint. Some of evaluators need model that supports chat completion.
+
+```python
+default_connection = project_client.connections.get_default(connection_type=ConnectionType.AZURE_OPEN_AI)
+```
+
+##### Example Remote Evaluation
+
+```python
+import os
+from azure.ai.projects import AIProjectClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import Evaluation, Dataset, EvaluatorConfiguration, ConnectionType
+from azure.ai.evaluation import F1ScoreEvaluator, RelevanceEvaluator, HateUnfairnessEvaluator
+
+
+# Create project client
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+
+# Upload data for evaluation and get dataset id
+data_id, _ = project_client.upload_file("<data_file.jsonl>")
+
+deployment_name = "<deployment_name>"
+api_version = "<api_version>"
+
+# Create an evaluation
+evaluation = Evaluation(
+    display_name="Remote Evaluation",
+    description="Evaluation of dataset",
+    data=Dataset(id=data_id),
+    evaluators={
+        "f1_score": EvaluatorConfiguration(
+            id=F1ScoreEvaluator.id,
+        ),
+        "relevance": EvaluatorConfiguration(
+            id=RelevanceEvaluator.id,
+            init_params={
+                "model_config": default_connection.to_evaluator_model_config(
+                    deployment_name=deployment_name, api_version=api_version
+                )
+            },
+        ),
+        "violence": EvaluatorConfiguration(
+            id=ViolenceEvaluator.id,
+            init_params={"azure_ai_project": project_client.scope},
+        ),
+    },
+)
+
+
+evaluation_response = project_client.evaluations.create(
+    evaluation=evaluation,
+)
+
+# Get evaluation
+get_evaluation_response = project_client.evaluations.get(evaluation_response.id)
+
+print("----------------------------------------------------------------")
+print("Created evaluation, evaluation ID: ", get_evaluation_response.id)
+print("Evaluation status: ", get_evaluation_response.status)
+if isinstance(get_evaluation_response.properties, dict):
+    print("AI Studio URI: ", get_evaluation_response.properties["AiStudioEvaluationUri"])
+print("----------------------------------------------------------------")
+```
+
+NOTE: For running evaluators locally refer to [Evaluate with the Azure AI Evaluation SDK][evaluators].
```

Comment:
Future work item: automatically grab code snippets from samples, using document tags and Azure tool, instead of hard-coded code.